File: Linguagem C Exercicios/C-Condicional/GLICOSE.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    double glicose = 0;


    printf("Digite a quantidade de glicose: ");
    scanf("%lf",&glicose);

    if(glicose <=100){
        printf("Classificacao: Normal");
    }else if(glicose <=140){
        printf("Classificacao: Elevado");
    }else{
        printf("Classificacao: Diabetes");
    };


    return 0;
}

File: Linguagem C#/Condicional/Glicose.cs
using System;
using System.Globalization;

namespace Glicose {
    class Program {
        static void Main(string[] args) {

            CultureInfo CI = CultureInfo.InvariantCulture;

            double glicose;

            Console.Write("Digite a medida da glicose: ");
            glicose = double.Parse(Console.ReadLine(),CI);

            if (glicose <= 100) {
                Console.WriteLine("Classificação: Normal");
            }else if (glicose <= 140) {
                Console.WriteLine("Classificaçao: Elevado");
            }else {
                Console.WriteLine("Classificação: Diabetes");
            };
        }
    }
}
